checker/jdk/nullness/src/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -0,0 +1,41 @@
+package java.util.concurrent;
+import org.checkerframework.dataflow.qual.Pure;
+import org.checkerframework.dataflow.qual.SideEffectFree;
+import java.util.*;
+import org.checkerframework.checker.nullness.qual.KeyFor;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class ConcurrentHashMap<K extends @NonNull Object, V extends @NonNull Object> extends AbstractMap<K, V>
+        implements ConcurrentMap<K, V>, java.io.Serializable 
+{
+
+     public ConcurrentHashMap(int initialCapacity,float loadFactor, int concurrencyLevel) {throw new RuntimeException("skeleton method");}

```

Comment:
Have spaces around the method bodies: `{ throw new RuntimeException("skeleton method"); }`